components/form/builder/src/Range/Default/index.js

```diff
@@ -0,0 +1,127 @@
+import {isValidElement, memo} from 'react'
+
+import PropTypes from 'prop-types'
+import {field, createComponentMemo} from '../../prop-types/index.js'
+
+import MoleculeInputField from '@s-ui/react-molecule-input-field'
+
+const joinInputValues = inputValues =>
+  inputValues ? Object.values(inputValues).join(',') : ''
+
+const splitInputValues = inputValues => {
+  const [from, to] = inputValues?.split(',') || []
+  return {
+    ...(from && {from}),
+    ...(to && {to})
+  }
+}
+
+const DefaultRange = ({
+  range,
+  tabIndex,
+  onChange,
+  onFocus,
+  onBlur,
+  errors,
+  alerts,
+  renderer
+}) => {
+  const fromInputId = `${range.id}From`
+  const toInputId = `${range.id}To`
+
+  const inputValues = splitInputValues(range.value)
+
+  const onChangeCallback = (ev, {value, name}) => {
+    if (name === fromInputId) {
+      onChange(range.id, joinInputValues({...inputValues, from: value}))
+    }
+    if (name === toInputId) {
+      onChange(range.id, joinInputValues({...inputValues, to: value}))
+    }
+  }
+
+  const onBlurCallback = ev => onBlur(ev.target.id)
+  const onFocusCallback = ev => onFocus(ev.target.id)
+
+  const rangeProps = {
+    id: range.id,
+    display: range.display,
+    value: range.value,
+    onChange: onChangeCallback,
+    onBlur: onBlurCallback,
+    onFocus: onFocusCallback,
+    ...(range.disabled && {
+      disabled: true
+    }),
+    ...(range.hidden && {
+      hidden: true
+    })
+  }
+
+  const fromInputProps = {
+    id: fromInputId,
+    name: fromInputId,
+    label: range.label || '',
+    value: inputValues?.from,
+    placeholder: 'Desde',
```

Comment:
Maybe it's interesting to use the placeholder from the field defined in the JSON instead of having it hardcoded here.

Reply:
The problem is that it's only one field